AutocompleteClient/FW/Config/ConstructorIOConfig.swift

```diff
@@ -1,6 +1,4 @@
-//
-//  ConstructorIOConfig.swift
-//  Constructor.io
+//config
```

Comment:
Why did we do this?

Reply:
🤔  Not sure why that happened. Will revert.